mmdet/models/dense_heads/yolox_head.py

```diff
@@ -212,6 +212,7 @@ def forward(self, feats):
                            self.multi_level_conv_reg,
                            self.multi_level_conv_obj)
 
+    @force_fp32(apply_to=('cls_scores', 'bbox_preds', 'objectnesses'))
```

Comment:
Is it appropriate to put this above `def _bboxes_nms(self, cls_scores, bboxes, score_factor, cfg):`

Reply:
good idea.

Reply:
I find fp16 tensor cannot do `.exp()` operation in cpu. And in cuda, fp16 tensor will autocast to fp32 after `exp()` in `_bbox_decode`. I consider Adding `force_fp32` above `get_bboxes` may avoids some potential errors.

Reply:
Referring https://pytorch.org/docs/stable/amp.html#ops-that-can-autocast-to-float32